### PR TITLE
Parameterize package name

### LINKED ISSUE #21: Package name
The package name is currently fixed at `com.lightbend.akka`, we might want to use the 'organization' for that.

### PR DESCRIPTION
**Pull request by jiminhsieh**
Fix #21

### CODE REVIEW

**Review by ktoso — APPROVED** [review]
Looks like a good improvement, thanks!

**Comment by ktoso** [comment]
cc @raboof I guess we might want to do the same in the scala one as well

**Comment by jiminhsieh** [comment]
I did the similar update at this PR https://github.com/akka/akka-http-quickstart-scala.g8/pull/41.

File: src/main/g8/default.properties [hunk 1]
```diff
@@ -2,3 +2,4 @@ name=My Akka HTTP Project
 akka_http_version=10.1.1
 akka_version=2.5.12
 organization=com.example
+package=$organization$
```
Comment (jiminhsieh) [inline src/main/g8/default.properties:5]: Although it's not a big thing, I am thinking of replacing the above two properties to:
```INI
organization=com.lightbend
package=$organization$.akka.http
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
-A [Giter8][g8] template for Akka HTTP in Java
+## Akka HTTP quickstart in Java
+
+Please see the [quickstart guide](https://developer.lightbend.com/guides/akka-http-quickstart-java/) for a
+walk through the code.
+
+You can use [Giter8][g8] to create your own project from the quickstart. Alternatively, you can download
+a packaged version of this project by clicking "Create a project for me!" on the
+[Lightbend Getting Started](https://developer.lightbend.com/start/?group=akka&project=akka-http-quickstart-java)
+page.
```
Comment (raboof) [inline README.md:9]: :+1:

File: src/main/g8/default.properties (hunk 1 above)
Comment (raboof) [reply src/main/g8/default.properties:5]: Hmm. For Java that makes sense, but for Scala projects the convention to have the organization in the package name is not so widespread... so perhaps good to keep it 'unstructured' to be consistent?

**Review by raboof — APPROVED** [review]
Excellent improvements!

**Pull request merged**